static/scss/product-details.scss

```
@@ -0,0 +1,218 @@
+// sass-lint:disable mixins-before-declarations
+.container {
+  max-width: 1260px;
+  padding: 0 15px;
+  box-sizing: border-box;
+  margin: 0 auto;
```

```suggestion
  box-sizing: border-box;
```

1. I don't think the `padding` or `margin` rules are necessary here. I disabled both in mobile and desktop widths and it doesn't seem to have an effect
2. If this page needs to have a max-width that is greater than the standard width for bootstrap `container` elements, then that should be applied to _all_ top-level `container` elements. I think we should hold off on setting a new max-width for all elements and see what Ferdi, Peter, and the designer have to say. That can be done in a separate PR